src/parser/content-evaluator-module.ts

```diff
@@ -65,13 +65,13 @@ export class ContentEvaluatorModule implements Module {
     const allComments: { id: number; comment: string; author: string }[] = [];
     for (const commentObj of allCommentsUnClean) {
       if (commentObj.user) {
-        allComments.push({ id: commentObj.id, comment: commentObj.body || "", author: commentObj.user.login });
+        allComments.push({ id: commentObj.id, comment: commentObj.body ?? "", author: commentObj.user.login });
       }
     }
 
     for (const key of Object.keys(result)) {
       const currentElement = result[key];
-      const comments = currentElement.comments || [];
+      const comments = currentElement.comments ?? [];
```

Comment:
This seems like a significant logic change. What's going on here?

Reply:
`sonar` throws errors on this because of type coalesce. I can disable the rule if you prefer.